### ISSUE TITLE
[MU4 Issue] Keyboard navigation in "New Score" dialog does not work for text entry fields in MacOS

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#9765]
**Describe the bug**
In MacOS, neither the **tab** nor the **up/down arrow** keys function to navigation text fields in the "New Score" dialog 

![tab-navigation-score-properties](https://user-images.githubusercontent.com/86290556/142194990-b12bceb8-d1b1-428a-9bd8-e39ffc66dabf.gif)

**Expected behavior**
_Ideally_, the **Tab** key should cycle between all text fields in this order:
1. Title
2. Composer
3. Subtitle
4. Lyricist
5. Copyright

This would be my strong preference. If this is not possible, however, then at least the **up/down arrow** keys should function to cycle between these text fields (as it currently does in Windows builds of MS4).

**Desktop (please complete the following information):**
 - MacOS 11.6

**Comment by cbjeukendrup** [comment]
Currently, you need to press ESC first, so that the cursor in the text field stops blinking, and then you can use the arrow keys to navigate between those text fields.

**Comment by bkunda** [comment]
Even worse than this, you have to press ESC _each time_ you want to move to the next field!
This should definitely not work this way. The ESC should not come into the navigation user flow at all.

**Comment by Eism** [comment]
It is strange... because I fixed it in https://github.com/musescore/MuseScore/pull/9523

**Comment by bkunda** [comment]
Sorry @Eism 😳. I have tested this in yesterday's nightly.

**Comment by mattblo** [comment]
Also in the latest Windows builds jumping to the next field does not work with TAB or arrow keys

**Comment by Eism** [comment]
@bkunda implemented down arrow navigation as it is in Windows and Linux
please, check #10746
